net-wireless/python-wifi/python-wifi-0.3.1-r2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/net-wireless/python-wifi/python-wifi-0.3.1-r1.ebuild,v 1.3 2012/02/25 11:53:52 patrick Exp $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Provides r/w access to a wireless network card's capabilities using the Linux Wireless Extensions"
+HOMEPAGE="http://pypi.python.org/pypi/python-wifi"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz
+	mirror://sourceforge/${PN}.berlios/${P}.tar.gz"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+LICENSE="LGPL-2.1 examples? ( GPL-2 )"
+IUSE="examples"
+
+RDEPEND=""
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+DOCS=( docs/AUTHORS docs/BUGS docs/DEVEL.txt docs/TODO )
+
+src_install() {
+	distutils-r1_src_install
+	use examples && doins -r examples
```

Comment:
Err, this is wrong. You can't `doins` something without using `insinto` first. And you likely want `dodoc -r` here anyway, since that's what puts files in `/usr/share/doc/${PF}` :).